includes/class-rest.php

```diff
@@ -422,8 +422,7 @@ private static function populate_original_post_status( $post ) {
 	 * @param WP_Post $post New post object.
 	 */
 	private static function populate_post_human_time( $post ) {
-		$post->post_human_time =  human_time_diff( get_the_time( 'U', $post ), current_time( 'timestamp' ) );
-		get_post_status( $original_post_id );
```

Comment:
I don't know if this get_post_status just happened to be here because of some strange merge thing... But it shouldn't be, right? Maybe it's missing from somewhere else @redundans?